src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php

```diff
@@ -0,0 +1,160 @@
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Templating\Tests\Storage\FileStorageTest;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class DownloadBuildCommand extends Command
+{
+    protected static $defaultName = 'sulu:admin:download-build';
+
+    /**
+     * @var HttpClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    /**
+     * @var string
+     */
+    private $suluVersion;
+
+    /**
+     * @var string
+     */
+    private $remoteRepository;
+
+    /**
+     * @var string
+     */
+    private $remoteArchive;
+
+    const ASSETS_DIR = DIRECTORY_SEPARATOR . 'assets' . DIRECTORY_SEPARATOR . 'admin' . DIRECTORY_SEPARATOR;
+
+    const BUILD_DIR = DIRECTORY_SEPARATOR . 'public' . DIRECTORY_SEPARATOR . 'build' . DIRECTORY_SEPARATOR . 'admin';
+
+    const REPOSITORY_NAME = 'skeleton';
+
+    public function __construct(HttpClientInterface $httpClient, string $projectDir, string $suluVersion)
+    {
+        parent::__construct();
+
+        $this->httpClient = $httpClient;
+        $this->projectDir = $projectDir;
+        $this->suluVersion = $suluVersion;
+        $this->remoteRepository = 'https://raw.githubusercontent.com/sulu/skeleton/' . $suluVersion;
+        $this->remoteArchive = 'https://codeload.github.com/sulu/skeleton/zip/' . $suluVersion;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Downloads the current admin application build from the sulu/skeleton repository.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln('<info>Checking for changed files...</info>');
+
+        $indexJs = static::ASSETS_DIR . 'index.js';
+        $packageJson = static::ASSETS_DIR . 'package.json';
+        $webpackConfigJs = static::ASSETS_DIR . 'webpack.config.js';
+
+        $localIndexJsHash = $this->getLocaleFileHash($indexJs);
+        $localPackageJsonHash = $this->getLocaleFileHash($packageJson);
+        $localWebpackConfigJsHash = $this->getLocaleFileHash($webpackConfigJs);
+
+        $remoteIndexJsHash = $this->getRemoteFileHash($indexJs);
+        $remotePackageJsonHash = $this->getRemoteFileHash($packageJson);
+        $remoteWebpackConfigJsHash = $this->getRemoteFileHash($webpackConfigJs);
+
+        if ($localIndexJsHash !== $remoteIndexJsHash
+            || $localPackageJsonHash !== $remotePackageJsonHash
+            || $localWebpackConfigJsHash !== $remoteWebpackConfigJsHash
+        ) {
+            throw new \Exception(
+                sprintf(
+                    'The files in the local "%s" folder do not match the ones in the remote repository "%s".' . PHP_EOL
+                    . 'Either the build has been modified, which means it has to be done manually with NPM or the files'
+                    . ' in your repository are outdated and have to be copied from the remote repository.',
```

Comment:
Maybe add also something like "If you have additional bundles or custom admin code you need to build the admin.js yourself"